features/shopcarts.feature

```diff
@@ -7,3 +7,15 @@ Scenario: The server is running
     When I visit the "home page"
     Then I should see "Shopcart Demo REST API Service"
     Then I should not see "404 Not Found"
+
+Scenario: List all shopcarts
+    When I visit "shopcarts"
+    Then I should see "Settlers of Catan"
```

Comment:
do we have any description of shopping carts in our data? I didn't see anything specific in the data